mslib/mscolab/server.py

```diff
@@ -62,14 +62,14 @@
 APP.config['UPLOAD_FOLDER'] = mscolab_settings.UPLOAD_FOLDER
 APP.config['MAX_CONTENT_LENGTH'] = mscolab_settings.MAX_UPLOAD_SIZE
 APP.config['SECRET_KEY'] = mscolab_settings.SECRET_KEY
-APP.config['SECURITY_PASSWORD_SALT'] = mscolab_settings.SECURITY_PASSWORD_SALT
-APP.config['MAIL_DEFAULT_SENDER'] = mscolab_settings.MAIL_DEFAULT_SENDER
-APP.config['MAIL_SERVER'] = mscolab_settings.MAIL_SERVER
-APP.config['MAIL_PORT'] = mscolab_settings.MAIL_PORT
-APP.config['MAIL_USERNAME'] = mscolab_settings.MAIL_USERNAME
-APP.config['MAIL_PASSWORD'] = mscolab_settings.MAIL_PASSWORD
-APP.config['MAIL_USE_TLS'] = mscolab_settings.MAIL_USE_TLS
-APP.config['MAIL_USE_SSL'] = mscolab_settings.MAIL_USE_SSL
+APP.config['SECURITY_PASSWORD_SALT'] = mscolab_settings.__dict__.get("SECURITY_PASSWORD_SALT", None)
```

Comment:
maybe briefer:

getattr(mscolab_settings, "SECURITY_PASSWORD_SALT", None) ?

Reply:
> maybe briefer:
> 
> getattr(mscolab_settings, "SECURITY_PASSWORD_SALT", None) ?

ah, will improve this in 6.0.1